roles/builder/tasks/main.yml

```diff
@@ -130,7 +130,8 @@
 
 - name: Create kickstart file
   ansible.builtin.template:
-    src: 'templates/kickstart.j2'
+    force: true
```

Comment:
force: true is the default, we can leave this line out and the file will still be overwritten.